tailwind.config.js

```diff
@@ -23,6 +24,29 @@ module.exports = {
         titillium: 'TitilliumWeb-Regular',
       },
     },
+    fontSize: {
```

Comment:
De ce nu folosim font sizes din tailwind? (https://tailwindcss.com/docs/font-size)

Reply:
Valorile din design nu am vazut sa fie deja predefinite in tailwind.